deployment/aws-k8s/example-runner/deployServicesToK8s.sh

```diff
@@ -44,12 +44,19 @@ fi
 
 # Begin script in case all parameters are correct
 helm dep up
+
+# Added extra params to ensure that AWS deploys use the shared one at 'palisade-shared' DNS
 helm upgrade --install --wait --atomic palisade . \
-    --namespace ${namespace} \
-    --set global.repository=${repository} \
-    --set global.hostname=${hostname} \
-    --set global.persistence.dataStores.palisade-data-store.aws.volumeHandle=${datastore} \
-    --set global.persistence.classpathJars.aws.volumeHandle=${classpathjars} \
-    --set global.deployment=example \
     --set global.hosting=aws \
-    --timeout 300s
+    --set global.repository="${repository}" \
+    --set global.hostname="${hostname}" \
+    --set global.persistence.dataStores.palisade-data-store.aws.volumeHandle="${datastore}" \
+    --set global.persistence.classpathJars.aws.volumeHandle="${classpathjars}" \
+    --set global.deployment=example \
+    --set global.kafka.install=false \
+    --set global.kafka.exports.fullnameOverride=kafka \
+    --set global.redis.install=false \
+    --set global.redis.exports.fullnameOverride=redis \
+    --set global.serviceName=.palisade-shared \
```

Comment:
Is the full-stop required after the `=` for the serviceName?

Reply:
Yes it's used as the separator for the FQDN. i.e. the name in this instance will be palisade-service.palisade-shared; however rather than add the . in code if it's here if the service changes and we need to make it  palisade-service-test.palisade-shared we can set the serviceName to -test.palisade-shared.